packages/model-viewer/src/three-components/gltf-instance/correlated-scene-graph.ts

```diff
@@ -284,7 +283,6 @@ export class CorrelatedSceneGraph {
             this.gltfElementMap.set(gltfElement, threeObjects);
           }
           threeObjects.add((object as Mesh).material as Material);
-          onUpdate();
```

Comment:
I removed this `onUpdate` function, because we don't need to request a new render on every visit to a scene node, plus is wasn't doing what we wanted anyway.